google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryRequestInfo.java

```diff
@@ -70,7 +70,9 @@ boolean isFastQuerySupported() {
         && config.getTableDefinitions() == null
         && config.getTimePartitioning() == null
         && config.getUserDefinedFunctions() == null
-        && config.getWriteDisposition() == null;
+        && config.getWriteDisposition() == null
+        && jobId.getJob() == null
```

Comment:
probably want a null check on jobId before checking the fields.

Reply:
https://github.com/googleapis/java-bigquery/blob/main/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobId.java#L48

GetJob return's the job's id

Reply:
Done